documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -0,0 +1,225 @@
+* To install on *macOS* or *Windows*, see the http://nmap.org/download.html[nmap.org download page].
+
+To use `nmap` to scan the devices on your network, you need to know the subnet you are connected to. First find your own IP address, in other words the one of the computer you're using to find your Raspberry Pi's IP address:
+
+* On *Linux*, type `hostname -I` into a terminal window
+* On *macOS*, go to `System Preferences` then `Network` and select your active network connection to view the IP address
+* On *Windows*, go to the Control Panel, then under `Network and Sharing Center`, click `View network connections`, select your active network connection and click `View status of this connection` to view the IP address
+
+Now you have the IP address of your computer, you will scan the whole subnet for other devices. For example, if your IP address is `192.168.1.5`, other devices will be at addresses like `192.168.1.2`, `192.168.1.3`, `192.168.1.4`, etc. The notation of this subnet range is `192.168.1.0/24` (this covers `192.168.1.0` to `192.168.1.255`).
+
+Now use the `nmap` command with the `-sn` flag (ping scan) on the whole subnet range. This may take a few seconds:
+
+[,bash]
+----
+nmap -sn 192.168.1.0/24
+----
+
+Ping scan just pings all the IP addresses to see if they respond. For each device that responds to the ping, the output shows the hostname and IP address like so:
+
+----
+Starting Nmap 6.40 ( http://nmap.org ) at 2014-03-10 12:46 GMT
+Nmap scan report for hpprinter (192.168.1.2)
+Host is up (0.00044s latency).
+Nmap scan report for Gordons-MBP (192.168.1.4)
+Host is up (0.0010s latency).
+Nmap scan report for ubuntu (192.168.1.5)
+Host is up (0.0010s latency).
+Nmap scan report for raspberrypi (192.168.1.8)
+Host is up (0.0030s latency).
+Nmap done: 256 IP addresses (4 hosts up) scanned in 2.41 seconds
+----
+
+Here you can see a device with hostname `raspberrypi` has IP address `192.168.1.8`. Note, to see the hostnames, you must run nmap as root by prepending `sudo` to the command.
+
+==== Getting IPv6 addresses by pinging from a second device 
+
+First find your own IP address(es), in other words the one of the computer you're using to find your Raspberry Pi's IP address
+by `hostname -I`
+
+`fd00::ba27:ebff:feb6:f293 2001:db8:494:9d01:ba27:ebff:feb6:f293`
+
+The example shows two IP addresses. The first one is a so called unique local unicast address(`fc00::/7`). The second one is the global unicast address(`2000::/3`). It is also possible to see only one of them depending on your network (router) configuration. Both addresses are valid for reaching the Raspberry Pi within your LAN. The address out of `2000::/3` is accessible world wide, provided your router's firewall is opened.
+
+Now use one of IPs from the first step to ping all local nodes:
+
+----
+ping -c 2 -I 2001:db8:494:9d01:ba27:ebff:feb6:f293  ff02::1
+ping -c 2 -I 2001:db8:494:9d01:ba27:ebff:feb6:f293  ff02::1%eth0
```

Comment:
Why are there two ping commands here?

Reply:
Honestly I'm not sure -- I left the command as-is because I don't fully understand it and I'm trying to limit scope creep :)

Reply:
Right; it was only towards the end of reviewing this that I realised that you were just moving existing content from one section to another, and not adding new content here :facepalm: